src/core/config/srp_client.h

```diff
@@ -218,4 +218,72 @@
 #define OPENTHREAD_CONFIG_SRP_CLIENT_RETRY_INTERVAL_GROWTH_FACTOR_DENOMINATOR 10
 #endif
 
+/**
+ * @def OPENTHREAD_CONFIG_SRP_CLIENT_SERVICE_POOL_ENABLE
+ *
+ * Define to 1 to enable SRP Client service pool feature.
+ *
+ */
+#define OPENTHREAD_CONFIG_SRP_CLIENT_SERVICE_POOL_ENABLE OPENTHREAD_CONFIG_SRP_CLIENT_ENABLE
+
+/**
+ * @def OPENTHREAD_CONFIG_SRP_CLIENT_SERVICE_POOL_MAX_SERVICES
+ *
+ * Specifies number of service entries in the SRP client service pool.
+ *
+ * This config is applicable/used only when `OPENTHREAD_CONFIG_SRP_CLIENT_SERVICE_POOL_ENABLE` is enabled.
```

Comment:
nit: reduce redundant words (here and below)
```suggestion
 * This config is applicable only when `OPENTHREAD_CONFIG_SRP_CLIENT_SERVICE_POOL_ENABLE` is enabled.
```